specs/latest/2.0/index.html

```diff
@@ -1073,20 +1170,32 @@ <h2 class="no-toc">Table of contents</h2>
         See <a href="#COPYING_BUFFERS">Copying Buffers</a> for restrictions imposed by the WebGL 2 API.
       </dd>
       <dt>
-        <p class="idl-code">void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData)
+        <p class="idl-code">
+          void getBufferSubData(GLenum target, GLintptr srcByteOffset,
+                                BufferDataSource? dstBuffer,
+                                optional unsigned long dstOffset = 0,
+                                optional unsigned long length = 0)
         </p>
       </dt>
       <dd>
-        For the WebGLBuffer bound to the passed target, read <code>returnedData.byteLength</code>
-        bytes from the buffer starting at byte offset <code>offset</code> and write them
-        to <code>returnedData</code>. If <code>offset + returnedData.byteLength</code> would extend
-        beyond the end of the buffer an <code>INVALID_VALUE</code> error is generated.
-        If <code>returnedData</code> is null then an <code>INVALID_VALUE</code> error is generated.
-        If <code>offset</code> is less than zero, an <code>INVALID_VALUE</code> error is generated.
-        If zero is bound to <code>target</code>, an <code>INVALID_OPERATION</code> error is generated.
-        If <code>target</code> is <code>TRANSFORM_FEEDBACK_BUFFER</code>, and any transform feedback object
-        is currently active, an <code>INVALID_OPERATION</code> error is generated.
-        If any error is generated, no data is written to <code>returnedData</code>.
+        If <code>length</code> is 0, it defaults to <code>dstBuffer.length - dstOffset</code>.
+        For the WebGLBuffer bound to the passed target, read <code>dstLength</code>
```

Comment:
dstLength needs to be defined -- or this reference needs to be changed to "length".